app/assets/javascripts/oxalis/model/reducers/skeletontracing_reducer_helpers.js

```diff
@@ -309,7 +312,8 @@ function splitTreeByNodes(
   let intermediateState = state;
   // For each new tree root create a new tree
   const cutTrees = _.compact(
-    newTreeRootIds.map((rootNodeId, index) => {
+    // Sort the treeRootIds, so the tree connected to the node with the lowest id will remain the original tree (treeId, name, timestamp)
+    newTreeRootIds.sort().map((rootNodeId, index) => {
```

Comment:
For the record (since my second comment got lost somewhere): `[1, 10, 3].sort()' evaluates to '[1, 10, 3]`, since `sort` without parameters does a lexicographical sort instead of a numeric one.